A32NX/html_ui/Pages/VCockpit/Instruments/NavSystems/LogicElements/A32NX_FlightPlanManager.js

```diff
@@ -654,7 +654,7 @@ class FlightPlanManager {
             callback();
         });
     }
-    get cruisingAltitude() {
+    getcruisingAltitude() {
```

Comment:
Why do you want to remove the getter? You can just refer to it as mcdu.cruisingAltitude (not a function call).

Reply:
true, makes no sense what I did there. fixed it, ty! 👍